### PR TITLE
Follow-ups for install command functionality

### PR DESCRIPTION
**Pull request by mwleeds**

### CODE REVIEW

**Comment by matthiasclasen** [comment]
I agree about the difficulty with one line. You _can_ describe variants of the command in multiple lines, but what you have here seems like an improvement, anyway.

File: app/flatpak-builtins-install.c [hunk 1]
```diff
@@ -369,28 +372,20 @@ flatpak_builtin_install (int argc, char **argv, GCancellable *cancellable, GErro
                     continue;
                   else
                     {
-                      if (!flatpak_resolve_duplicate_remotes (dirs, this_remote, &dir_with_remote, cancellable, error))
-                        return FALSE;
-
-                      if (!opt_yes &&
-                          !flatpak_yes_no_prompt (TRUE, /* default to yes on Enter */
-                                                  _("Found refs similar to ‘%s’ in remote ‘%s’ (%s).\nIs that the remote you want to use?"),
-                                                  argv[1], this_remote, flatpak_dir_get_name (dir_with_remote)))
```
Comment (matthiasclasen) [inline app/flatpak-builtins-install.c]: I guess this makes the default-to-yes feature we added for this no longer used, but nevermind.

File: app/flatpak-builtins-utils.h [hunk 2]
```diff
@@ -30,6 +30,16 @@
 /* Appstream data expires after a day */
 #define FLATPAK_APPSTREAM_TTL 86400
 
+typedef struct RemoteDirPair
+{
+  gchar              *remote_name;
```
Comment (matthiasclasen) [inline app/flatpak-builtins-utils.h]: I think we are generally using g-less types in flatpak when possible, so char *

File: app/flatpak-builtins-install.c (hunk 1 above)
Comment (mwleeds) [reply app/flatpak-builtins-install.c]: It's still used in other prompts though

**Comment by matthiasclasen** [comment]
@rh-atomic-bot: r+

**Comment by matthiasclasen** [comment]
Looks like this needs some test fixes

**Comment by mwleeds** [comment]
Looks like I fixed most but not all of the tests :/

File: app/flatpak-builtins-utils.h (hunk 2 above)
Comment (mwleeds) [reply app/flatpak-builtins-utils.h]: I just copied this from elsewhere but if we want to enforce that norm we could do a replace-all across the codebase

**Comment by matthiasclasen** [comment]
@rh-atomic-bot: r+

**Pull request closed without merging**